Formula/l/llvm.rb

```diff
@@ -440,15 +446,35 @@ def install
   end
 
   def caveats
+    s = <<~EOS
+      `lld` is now provided in a separate formula:
+        brew install lld
+    EOS
+
     on_macos do
-      <<~EOS
-        To use the bundled libc++ please add the following LDFLAGS:
-          LDFLAGS="-L#{opt_lib}/c++ -L#{opt_lib} -lunwind"
+      s += <<~EOS
+
+        To use the bundled libunwind please use the following LDFLAGS:
+          LDFLAGS="-L#{opt_lib}/unwind -lunwind"
+
+        To use the bundled libc++ please use the following LDFLAGS:
+          LDFLAGS="-L#{opt_lib}/c++ -L#{opt_lib}/unwind -lunwind"
+
+        #{Tty.bold}NOTE:#{Tty.reset} You probably want to use the libunwind and libc++ provided by macOS unless you know what you're doing.
```

Comment:
Does `formulae.brew.sh` handle these ANSI escape codes? 
https://github.com/Homebrew/formulae.brew.sh/blob/master/_layouts/formula.html#L207

Reply:
Not sure. Dropped it for now.